mlflow/server/js/src/utils/Utils.js

```diff
@@ -107,11 +112,42 @@ class Utils {
         res = <a href={url}>{res}</a>;
       }
       return res;
+    } else if (run.source_type === "NOTEBOOK") {
+      const notebookIdTag = 'mlflow.databricks.notebookID';
+      const webappUrlTag = 'mlflow.databricks.webappURL';
+      const notebookId = tags && tags[notebookIdTag] && tags[notebookIdTag].value;
+      const webappUrl = tags && tags[webappUrlTag] && tags[webappUrlTag].value;
+      if (notebookId && webappUrl) {
+        res = (<a title={run.source_name} href={`${webappUrl}/#notebook/${notebookId}`}>
```

Comment:
Instead of displaying the full notebook path here, just display the last part (after the last slash).